test/unit/borealis_antennas_iq_data_sets.py

```diff
@@ -0,0 +1,217 @@
+"""
+Test data sets for Borealis Bfiq.
+"""
+
+from collections import OrderedDict
+from numpy import array, int16, int64, uint32, float32, float64, \
```

Comment:
Wouldn't it just be cleaner to `import numpy as np`, making all of these data types clearer down below?